samples/boards/stm32/power_mgmt/blinky/boards/nucleo_wb55rg.conf

```diff
@@ -0,0 +1,9 @@
+# set the CONFIG_SYS_CLOCK_TICKS_PER_SEC depends on the LPTIM prescaler
+# with a LPTIM1 prescaler of <1> to <8>, set the CONFIG_SYS_CLOCK_TICKS_PER_SEC to 4096
+# with a LPTIM1 prescaler of <16>, set the CONFIG_SYS_CLOCK_TICKS_PER_SEC to 2048
+# with a LPTIM1 prescaler of <32>, set the CONFIG_SYS_CLOCK_TICKS_PER_SEC to 1024
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1024
+
+# also set the CONFIG_STM32_LPTIM_CLOCK_LSE
+# to match the <clocks> property of the lptim1 node in the DTS
+CONFIG_STM32_LPTIM_CLOCK_LSE=y
```

Comment:
If LSE is configured in board.dts. `CONFIG_STM32_LPTIM_CLOCK_LSE=y` should also be part of default board configuration. 
Which is the case based on previous commit. This line could be cleaned up.

Reply:
see  https://github.com/zephyrproject-rtos/zephyr/pull/64067